libs/ngxtension/create-injectable/src/create-injectable.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { inject } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { createInjectable } from './create-injectable';
+
+describe(createInjectable.name, () => {
+	it('should be able to access property returned from injectable', () => {
+		let count = 0;
+		const MyInjectable = createInjectable(
+			() => {
+				count += 1;
+				return { someProp: 1 };
+			},
+			{ providedIn: 'root' },
+		);
+
+		TestBed.runInInjectionContext(() => {
+			// should be lazy until `inject()` is invoked
+			expect(count).toEqual(0);
+			const service = inject(MyInjectable);
```

Comment:
Maybe it could be useful to inject `MyInjectable` two times... and then check that count does not increment anymore after the first injection.

Then we are quite sure that it is a Singleton service.

Same thing could be done for the non-root injectable, but there we would expect that the count is incremented for every injection.